.github/actions/free-disk-space/action.yml

```diff
@@ -16,34 +16,13 @@
 # License-Filename: LICENSE
 
 name: "Free Disk Space"
-description: "Remove unneeded preinstalled Docker images and software to free disk space."
+description: "Remove unneeded software to free disk space."
 author: "The ORT Project Authors"
 
 runs:
   using: "composite"
 
   steps:
-    - name: Print Disk Space
```

Comment:
Shouldn't this removal go to a separate commit?

Reply:
The step is still there at line 26, actually I removed the step below the Docker image cleanup which was there to show how much disk space it made free, but Git shows it differently.